x-pack/solutions/security/plugins/security_solution/public/detection_engine/rule_management/components/rule_details/three_way_diff/comparison_side/translations.ts

```diff
@@ -93,3 +93,17 @@ export const MY_ORIGINAL_CHANGES_EXPLANATION = i18n.translate(
     },
   }
 );
+
+export const COMPARISON_SIDE_ERROR_FALLBACK_TITLE = i18n.translate(
+  'xpack.securitySolution.detectionEngine.rules.upgradeRules.comparisonSideErrorFallbackTitle',
+  {
+    defaultMessage: 'Unable to load comparison side',
```

Comment:
nit: Should it be called "Diff view", since user doesn't see "comparison side" anywhere in UI